src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -43,14 +42,9 @@ const SetAclPermissionForm = () => {
     setDocType(event.target.value);
   };
 
-  const clearInputFields = () => {
-    dispatch({ type: 'CLEAR_PROCESSING' });
```

Comment:
Since we're removing the old dispatch call, we can replace it with the set function from `useState`, something like `setProcessing(false)`.

The same would be applied for all the other dispatch calls being removed.

Reply:
I made this change.